Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorGrowFromSeedsEffect.py

```diff
@@ -37,21 +39,21 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Growing segments to create complete segmentation<br>.
-Location, size, and shape of initial segments and content of source volume are taken into account.
-Final segment boundaries will be placed where source volume brightness changes abruptly. Instructions:<p>
-<ul style="margin: 0">
-<li>Use Paint or other offects to draw seeds in each region that should belong to a separate segment.
-Paint each seed with a different segment. Minimum two segments are required.</li>
-<li>Click <dfn>Initialize</dfn> to compute preview of full segmentation.</li>
-<li>Browse through image slices. If previewed segmentation result is not correct then switch to
-Paint or other effects and add more seeds in the misclassified region. Full segmentation will be
-updated automatically within a few seconds</li>
-<li>Click <dfn>Apply</dfn> to update segmentation with the previewed result.</li>
-</ul><p>
-If segments overlap, segment higher in the segments table will have priority.
-The effect uses <a href="http://interactivemedical.org/imic2014/CameraReadyPapers/Paper%204/IMIC_ID4_FastGrowCut.pdf">fast grow-cut method</a>.
-<p></html>"""
+        return _("""<html>Growing segments to create complete segmentation<br>.
+        Location, size, and shape of initial segments and content of source volume are taken into account.
+        Final segment boundaries will be placed where source volume brightness changes abruptly. Instructions:<p>
+        <ul style="margin: 0">
+        <li>Use Paint or other offects to draw seeds in each region that should belong to a separate segment.
+        Paint each seed with a different segment. Minimum two segments are required.</li>
+        <li>Click <dfn>Initialize</dfn> to compute preview of full segmentation.</li>
+        <li>Browse through image slices. If previewed segmentation result is not correct then switch to
+        Paint or other effects and add more seeds in the misclassified region. Full segmentation will be
+        updated automatically within a few seconds</li>
+        <li>Click <dfn>Apply</dfn> to update segmentation with the previewed result.</li>
+        </ul><p>
```

Comment:
<p> here is for adding some space after the text (before the `Hide details.` link).
Closing paragraph tag `</p>` is optional in HTML. Adding it would make the text (and all translations) a little bit more complex.